### PR TITLE
Benjamimo1 fix/add css qc

### PR DESCRIPTION
**Pull request by benjamimo1**
new attempt at previous changes requested. Adds CSS styling to the quality checks static page.

### CODE REVIEW

**Comment by benjamimo1** [comment]
Current web inspector view:
image

File: pootle/templates/help/quality_checks.html [hunk 1]
```diff
@@ -3,19 +3,35 @@
 
 {% block header_block %}{% endblock %}
 
+{% block css_overrides %}
+{{
```
Comment (iafan) [inline pootle/templates/help/quality_checks.html:7]: Why the `{{` and `}}` wrappers?

**Comment by benjamimo1** [comment]
@iafan any input on the checks status?

**Comment by iafan** [comment]
@benjamimo1 the PR looks good. Let's merge it after the CI build is green.

**Pull request merged**